radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java

```diff
@@ -32,6 +36,9 @@
 	private final Object lock = new Object();
 	private final Map<RadixAddress, Boolean> syncedMap = new HashMap<>();
 
+	private final Map<String, Atom> stagedAtoms = new ConcurrentHashMap<>();
```

Comment:
Why are we using atoms here? Just as a container for particle groups + timestamp?